### ISSUE TITLE
gulp error on install 

### ISSUE THREAD

**Issue by umbz** [twostairs/paperwork#713]
Hi , when i launch gulp , i obtain this error: 

```
stream.js:94
      throw er; // Unhandled stream error in pipe.
            ^
Error: EACCES, open '/var/www/paperwork/frontend/public/css/freqselector.min.css'
    at Error (native)
```

**Comment by kuc** [comment]
`EACCES`? Do you have read permissions to `/var/www/paperwork/frontend/public/css/freqselector.min.css` file?

**Comment by jsEveryDay** [comment]
Yes it has to be a permissions issue.

That's why when you download, you dont git clone or download the zip...
Get the **TARball**
tar inherits permissions.

**Comment by Liongold** [comment]
The recommended way to install Paperwork is using git clone. Is this fixed?

**Comment by umbz** [comment]
Yes, it is

**Comment by codejockey1** [comment]
Having the same issue installing on a Raspberry PI using both the tarball and git clone. Any ideas?
/var/www/paperwork/frontend/public/css/ only has boostrap-editible.css and mathquill.css in it. I've tried this multiple times cloning from git and finally with tarball as recommended above.

**Comment by Liongold** [comment]
@codejockey1 Can you temporarily assign more permissions to the paperwork folder while installing? Would it work?

**Comment by codejockey1** [comment]
That seems to have gotten me past the missing files and any install errors, but now I'm stuck in the in-browser setup flow. I responded to a related issue here: https://github.com/twostairs/paperwork/issues/689

**Comment by Liongold** [comment]
Thus, for future reference, the solution for this problem is to temporarily assign more permissions to the paperwork folder during installation.

**Comment by jsEveryDay** [comment]
I do remember having permission issues but then I realized it was due to the initial files on this repo, everything is unecessary, you only need 1 folder **frontend**

This file https://github.com/twostairs/paperwork/blob/master/.tx/config pings transifex.com
Try my fork, it may work.

As far as permissions, I have all files running at 644 and php files at 664.
I never really changed them as they cam out of the tarball. All good.

@Liongold That's definitely the way to go, for the initial setup if you face any issues, change all php's to 755 then after setup go back to 644. If the 755 doesn't work then you most likely have an Nginx problem. Check the nginx logs, sometimes nginx is tough to get it right.
